### DISCUSSION TITLE
XPath values syntax highlighting

### DISCUSSION THREAD

**Discussion by danielHalan** [redhat-developer/vscode-yaml#1002] [Q&A]
Is it possible using configuration, define values in a Yaml file to use XPath syntax highlighting?

i.e. 
test: //div[@id='hello']

**Comment by gorkem** [comment]
No, highlighting is done with the standard vscode highlighting engine. Semantic highlighting is not implemented on vscode-yaml which might have been a step closer to what you are asking.
